Use usvg instead of resvg, minor fixes

zen3ger opened the pull request:
Typo and zoom control char fixes.
Added option to set msaa from commandline.
Replace resvg with usvg.

Closes #349

zen3ger commented:
I saw that the `cli` uses `clap`, I thought it might be an overkill here, but should I use it?

nical commented on examples/svg_render/src/main.rs:
```diff
@@ -240,10 +264,10 @@ fn update_inputs(scene: &mut Scene, event_loop: &mut glutin::EventsLoop) -> bool
                 VirtualKeyCode::Escape => {
                     status = false;
                 }
-                VirtualKeyCode::PageDown => {
```
I'd prefer to keep PgUp/down to control the zoom like in the other examples and the cli app.

zen3ger commented:
Sorry, I saw the `println!()` with sqare brackets and thought it was messed up. I fixed the text and added `clap`. If everything is fine, I'll squash the commits.
